src/couch_views/src/couch_views_indexer.erl

```diff
@@ -541,3 +556,25 @@ key_size_limit() ->
 
 value_size_limit() ->
     config:get_integer("couch_views", "value_size_limit", ?VALUE_SIZE_LIMIT).
+
+
+update_tasks(NumChanges, false) ->
+    [Changes] = couch_task_status:get([changes_done]),
+    Changes2 = Changes + NumChanges,
+    couch_task_status:update([{changes_done, Changes2}]);
+
+update_tasks(NumChanges, LastSeq) ->
+    [Changes] = couch_task_status:get([changes_done]),
+    Changes2 = Changes + NumChanges,
+    VS = fabric2_fdb:seq_to_vs(LastSeq),
+    VS1 = convert_version_stamp(VS),
+    couch_task_status:update([{changes_done, Changes2},
+        {current_version_stamp, VS1}]).
+
+
+convert_version_stamp({_, Stamp, _, DocNumber}) ->
+    VS = integer_to_list(Stamp) ++ "-" ++ integer_to_list(DocNumber),
+    list_to_binary(VS);
+
+convert_version_stamp(_) ->
```

Comment:
What would cause us to reach this state?

Reply:
initially, when we create the versionstamp it's `not_found`:
`ViewVS = couch_views_fdb:get_creation_vs(TxDb, Mrst),`